Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,140 @@
-hi
\ No newline at end of file
+### The html tag: 
+The <html> tag is used to specify that the text should start being interpreted as html code.
+
+### The head tag: 
+The head of an html code contains information related to the document, for example: it's title, link, style, etc. 
+
+```html 
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+      <link rel="stylesheet" type="text/css" href="gwoc_style.css">
+   </head>
+</html>
+```
+
+### Title: 
+A title tag specifies a title in your browser window.  
+```html
+<title>Hello! Welcome to Girlscript Winter of Contributing</title>
+```
+
+### Style and Link tag:
+The style tag contains information on how html elements should render in a browser.
+```html
+<html>
+   <head>
+      <style>
+         body {
+         background: yellow;
+         }
+         h1 {
+         color: green;
```

Comment:
It will be even more interesting if you can add a screenshot of the output too along with the code.

Reply:
added